src/scripts/main.js

```diff
@@ -1,3 +1,79 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const tableHead = document.querySelector('thead');
+  const tableBody = document.querySelector('tbody');
+  let sortLowToHigh = true; // true - low to high, false- high to low
```

Comment:
Consider using a more descriptive variable name instead of `sortLowToHigh`, such as `sortAscending`. This will improve code readability.